### ISSUE TITLE
`cargo doc` doesn't work in this crate

### ISSUE THREAD

**Issue by steveklabnik** [steveklabnik/rustdoc#22]
I'd like to generate docs for `rustdoc`, but it's blocked on https://github.com/rust-lang/cargo/issues/4283

**Comment by aochagavia** [comment]
With that issue closed as WONTFIX, are there any alternatives to make this work on Windows?

**Comment by steveklabnik** [comment]
@aochagavia that's a good question; I'm not actually sure.

It may also be something that we can't fix until this becomes the "official" rustodoc. Hrm.

**Comment by steveklabnik** [comment]
Looks like this is not necessarily WONTFIX.
